test/test-framework/pkg/apis/cache/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,5 +1,21 @@
 // +build !ignore_autogenerated
 
+/*
```

Comment:
Try running deepcopy again. Headers aren't generated any more unless you pass a header file to `deepcopy-gen`.